catalogue_pipeline/id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/IdMinterFeatureTest.scala

```diff
@@ -72,55 +53,60 @@ class IdMinterFeatureTest
     }
   }
 
-  it(
-    "mints the same ID for SourcedWorks that have matching source identifiers") {
+  it("mints the same IDs where source identifiers match") {
     withLocalSqsQueue { queue =>
       withLocalSnsTopic { topic =>
-        withIdentifiersDatabase { dbConfig =>
-          val flags = sqsLocalFlags(queue) ++ snsLocalFlags(topic) ++ dbConfig.flags
-
-          withServer(flags) { _ =>
-            eventuallyTableExists(dbConfig)
-
-            val miroID = "M0001234"
-            val title = "A limerick about a lion"
-
-            val identifier =
-              SourceIdentifier(
-                IdentifierSchemes.miroImageNumber,
-                "Work",
-                miroID)
-
-            val work = UnidentifiedWork(
-              title = Some(title),
-              sourceIdentifier = identifier,
-              version = 1,
-              identifiers = List(identifier))
-
-            val sqsMessage = SQSMessage(
-              Some("subject"),
-              toJson(work).get,
-              "topic",
-              "messageType",
-              "timestamp"
-            )
-
-            val messageCount = 5
-
-            (1 to messageCount).foreach { _ =>
-              sqsClient.sendMessage(
-                queue.url,
-                toJson(sqsMessage).get
-              )
-            }
+        withLocalS3Bucket { bucket =>
+          withIdentifiersDatabase { dbConfig =>
+            val flags =
+              sqsLocalFlags(queue) ++
+                snsLocalFlags(topic) ++
```

Comment:
Do we need snsLocalFlags AND messagingLocalFlags?

Reply:
Yes, it does for now, because this branch is not changing the id_minter - ingestor boundary. It should be removed in the other branch

Reply:
But the ingestor does not need those flags, so I'm taking them out from there